src/App.js

```diff
@@ -45,23 +45,23 @@ const App = () => {
   }, []);
 
   function showConfirm(user) {
-    setDeleteUser(user)
-    setShow(true)
+    setDeleteUser(user);
+    setShow(true);
   }
 
   function deleteItem() {
-    setItems(items.filter((item) => item.name !== deleteUser));
-    setShow(false)
+    setItems(items.filter((item) => item !== deleteUser));
```

Comment:
When comparing objects, best to use `Object.is(object1, object2)`

Reply:
But ideally, you would just be using the `index` anyway to check which item to delete, not the entire object.